src/views/ModelPlan/TaskList/Beneficiaries/BeneficiaryIdentification/index.tsx

```diff
@@ -0,0 +1,512 @@
+                                name="beneficiaries-dual-eligibility-how"
+                              />
+                            </FieldGroup>
+                          )}
+                          <Field
+                            as={Radio}
+                            id="beneficiaries-dual-eligibility-no"
+                            name="treatDualElligibleDifferent"
+                            label={h('no')}
+                            value="FALSE"
+                            checked={
+                              values.treatDualElligibleDifferent === 'NO'
+                            }
+                            onChange={() => {
+                              setFieldValue(
+                                'treatDualElligibleDifferent',
+                                'NO'
+                              );
+                            }}
+                          />
+                          <Field
+                            as={Radio}
+                            id="beneficiaries-dual-eligibility-tbd"
+                            name="treatDualElligibleDifferent"
+                            label={t('beneficiariesOptions.na')}
+                            value="TBD"
+                            checked={
+                              values.treatDualElligibleDifferent === 'TBD'
+                            }
+                            onChange={() => {
+                              setFieldValue(
+                                'treatDualElligibleDifferent',
+                                'TBD'
+                              );
+                            }}
+                          />
+                        </Fieldset>
+                        <AddNote
+                          id="beneficiaries-dual-eligibility-note"
+                          field="treatDualElligibleDifferentNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="excludeCertainCharacteristics"
+                        error={!!flatErrors.excludeCertainCharacteristics}
+                        className="margin-y-4 margin-bottom-8"
+                      >
+                        <Label htmlFor="beneficiaries-exclude">
+                          {t('excluded')}
+                        </Label>
+                        <FieldErrorMsg>
+                          {flatErrors.excludeCertainCharacteristics}
+                        </FieldErrorMsg>
+                        <Fieldset>
+                          <Field
+                            as={Radio}
+                            id="beneficiaries-exclude"
+                            name="excludeCertainCharacteristics"
+                            label={h('yes')}
+                            value="TRUE"
+                            checked={
+                              values.excludeCertainCharacteristics === 'YES'
```

Comment:
I think these can be stricter with something like `values.excludeCertainCharacteristics === TriStateAnswer .YES`.  Same for other instances